compiler/damlc/daml-prim-src/GHC/Base.daml

```diff
@@ -21,24 +21,35 @@ import GHC.Integer.Type() -- required so Integer optimisation works
 import GHC.Prim
 import GHC.Types
 
+-- | Used as an alternative in conditions.
 otherwise : Bool
 otherwise = True
 
+-- | HIDE
 getTag : a -> Int#
 getTag = magic @"getTag"
 
 -- This is referenced from error messages generated by ghc.
+-- | Concatenate two lists.
 (++) : Text -> Text -> Text
 (++) = primitive @"BEAppendText"
 infixr 5 ++
 
+-- | Takes a function from values of type `a` to values of type `b`, 
```

Comment:
Isn't something like

> `map f xs` applies `f` to all elements of `xs` and returns the list of results (in the same order as `xs`).

easier to understand?

Reply:
Much better, thanks!